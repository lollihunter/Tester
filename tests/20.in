17 10 20
1 1
5 8
7 3
3 8
2 3
2 9
7 2
2 8
12 8
13 5
9 4
3 7
9 3
4 6
12 6
13 7
9 1
10 1
11 6
17 10
