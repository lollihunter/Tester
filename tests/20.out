9
1 5 14 3 11 19 15 16 20 
